docs/tutorials/infer_phys_pars.rst

```diff
@@ -0,0 +1,503 @@
+*****************************
+Inferring physical parameters
+*****************************
+
+This tutorial describes how to infer physical parameters from the free
+parameters of a phenomenological model for the scintillation velocities of a
+pulsar on a circular orbit whose radiation is scattered by a single
+one-dimensional screen. The tutorial builds upon a :doc:`preceding tutorial
+<fit_velocities>` in which such a phenomenological model is fit to a time
+series of scintillation velocities.
+
+Further explanations and derivations of the equations seen here can be found in
+`Marten's scintillometry page
+<http://www.astro.utoronto.ca/~mhvk/scintillometry.html#org5ea6450>`_
+and Daniel Baker's "`Orbital Parameters and Distances
+<https://eor.cita.utoronto.ca/images/4/44/DB_Orbital_Parameters.pdf>`_"
+document. As in that document, the practical example here uses the parameter
+values for the pulsar PSR J0437-4715 as derived by `Reardon et al. (2020)
+<https://ui.adsabs.harvard.edu/abs/2020ApJ...904..104R/abstract>`_.
+
+The combined codeblocks in this tutorial can be downloaded as a Python script
+and as a Jupyter notebook:
+
+:Python script:
+    :jupyter-download:script:`infer_phys_pars.py <infer_phys_pars>`
+:Jupyter notebook:
+    :jupyter-download:notebook:`infer_phys_pars.ipynb <infer_phys_pars>`
+
+Preliminaries
+=============
+
+Imports.
+
+.. jupyter-execute::
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+
+    from astropy import units as u
+    from astropy import constants as const
+
+    from astropy.coordinates import Angle, SkyCoord, BarycentricMeanEcliptic
+
+    from astropy.visualization import quantity_support
+
+Set up support for plotting Astropy's
+:py:class:`~astropy.units.quantity.Quantity` objects, and make sure that the
+output of plotting commands is displayed inline (i.e., directly below the code
+cell that produced it).
+
+.. jupyter-execute::
+
+    quantity_support()
+
+    %matplotlib inline
+
+Set known parameters
+====================
+
+Set the pulsar system's coordinates :math:`(\alpha, \delta)`
+and proper motion components :math:`(\mu_{\alpha\ast}, \mu_\delta)`,
+as well as some of the system's parameters that are known from timing studies:
+its orbital period :math:`P_\mathrm{b}`, projected semi-major axis
+:math:`a_\mathrm{p} \sin( i_\mathrm{p} )`, and radial-velocity amplitude
+:math:`K_\mathrm{p} = 2 \pi a_\mathrm{p} \sin( i_\mathrm{p} ) / P_\mathrm{b}`.
+
+.. jupyter-execute::
+
+    psr_coord = SkyCoord('04h37m15.99744s -47d15m09.7170s')
+    mu_alpha_star = 121.4385 * u.mas / u.yr
```

Comment:
Added the proper motion to the ``SkyCoord``.

I specifically left the distance out because my intention with the example in the tutorial was to show the possible constraints when the distance is not known. I thought including the known distance could cause some confusion, but if you disagree we could add it.